catalyst_voices/lib/pages/registration/registration_stage_navigation.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:catalyst_voices_shared/catalyst_voices_shared.dart';
+import 'package:flutter/material.dart';
+
+class BackNextNavigation extends StatelessWidget {
```

Comment:
RegistrationBackNextNavigation? you called the file name after registration

Reply:
Well, yes. I can see that.

Reply:
fixed